src/include/kernel/abi.h

```diff
@@ -29,7 +29,7 @@
 
 /** \brief SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
-#define SOF_ABI_MINOR 29
+#define SOF_ABI_MINOR 30
```

Comment:
So this is not needed as this ABI versioning applies to IPC3 builds.

We still don't have infra to describe IPC4 infra. As this has been pending for long, let me file a ticket -> https://github.com/thesofproject/sof/issues/9080